libsolutil/CommonData.h

```diff
@@ -232,6 +232,17 @@ std::set<K> keys(std::map<K, V> const& _map)
 	return applyMap(_map, [](auto const& _elem) { return _elem.first; }, std::set<K>{});
 }
 
+/// @returns a pointer to the entry of @a _map at @a _key, if there is one, and nullptr otherwise.
+template<typename MapType, typename KeyType>
+auto fetchValue(MapType&& _map, KeyType const& _key)
```

Comment:
name: `getOptional`? `get`?

Reply:
``get`` will probably easily conflict with ``set::get``... ``getOptional`` sounds like it'd return an optional, not a pointer... but I'd be fine with it :-)...

``fetchValueIfPresent``... ``getMapValueIfAny``... I don't know, I hate naming things like this :-).

Reply:
`getMapValueIfAny` sounds good, or maybe even shorter `getValueIfAny`

Reply:
+1 for `getValueIfAny()`. My other suggestions would be `valueOrNull()` or `atOrNull()`.

Reply:
``valueOrNull`` doesn't sound bad actually :-).